### DISCUSSION TITLE
Bug Admin Product Price Input - The system automatically removes all zeros after the dot

### DISCUSSION THREAD

**Discussion by balanh-development** [swellstores/community#301] [Help] [answered]
Whenever we type ex: 300.000 your system always trim it to 300 when I click save.

![image](https://github.com/swellstores/community/assets/120219638/c87046f1-877d-475d-92f6-081e9e485e9c)

It still occurs in the ex below, the price after I save is 3.

![image (1)](https://github.com/swellstores/community/assets/120219638/cbc5abcd-3771-4278-92a2-03d9c41d242e)

I use VND as the main currency and it occurs in any input prices in products tab, promotions and so on.

I reported a bug 2 months ago via email but haven't received a response yet.

**Comment by steven-k1** [comment]
Hello @balanh-development, depending on the currency used, a comma or decimal point would be used based on the international currency format.

I tested with the currency shown in your screenshots, Vietnamese Dong (VND), and a comma appears to work as expected.

![image](https://github.com/swellstores/community/assets/77881204/b931ba45-8e6a-4419-b760-2fa20eba6736)

Additionally, VND is set up by default to not have any decimal digits, but this can be changed via the settings where you selected the currency.

![image](https://github.com/swellstores/community/assets/77881204/2f7092aa-ea1d-4660-83ad-0a42e6aea064)

**Reply by balanh-development** [reply]
https://github.com/swellstores/community/assets/120219638/be305558-3fe8-4488-a1ed-de38cfde5487

![image](https://github.com/swellstores/community/assets/120219638/7e6377d2-cf42-4a64-ad5d-0a6d02f8396b)

Here my currency setting, I made a video about the problem, it's happening in my store now, please check again.

**Reply by steven-k1** [reply, marked as answer]
Hello @balanh-development, From the screenshots provided, I can see that the 'Decimal digits' setting has not been set. As mentioned in my previous reply, VND is set up by default to not have any decimal digits, so this would need to be updated by setting this to the desired decimal point.

![Example 1](https://github.com/swellstores/community/assets/77881204/377e4edc-5775-4a0e-840d-b0d352902371)

![Example 2](https://github.com/swellstores/community/assets/77881204/5c4b9705-db90-4d13-82cf-33f26388ee34)

**Reply by balanh-development** [reply]
I set "Decimal digits": 3, I can set the value and get it back as I want, but ui acts very strange. When I mouse out the box, your system automatically append .000 after the value and I click save it gone. 

https://github.com/swellstores/community/assets/120219638/79939be9-70d1-41e0-b742-e9263d228087

**Reply by steven-k1** [reply]
@balanh-development Thanks for the details. I'll look into this further.

**Comment by balanh-development** [comment]
It's been almost a year, the bug still hasn't been fixed, do you have a plan to fix it? @steven-k1
